README.md

```diff
@@ -245,6 +245,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Exspanse](https://exspanse.com) - MLOPS Platform to build, train and deploy ML models and AI solutions. Free plan gives ability to create unlimited projects, 5Gb of cloud storage, 5 docker container images.
   * [ExtendsClass](https://extendsclass.com/rest-client-online.html) - Free web-based HTTP client to send HTTP requests.
   * [Export SDK](https://exportsdk.com) - PDF generator API with drag and drop template editor that also provides an SDK and no-code integrations. Free plan comes with 250 pages per month, unlimited users and three templates.
+  * [fakerjs.dev](https://fakerjs.dev/) - Generate massive amounts of fake (but realistic) data for testing and development.
```

Comment:
Please describe the SaaS offering and what is included in the free tier.

Reply:
@ripienaar **Faker.js** is an open-source JavaScript library that generates realistic fake data for various purposes, primarily for testing and prototyping applications. It allows developers to easily generate random names, addresses, phone numbers, email addresses, and other types of data.

Reply:
Ok. So this is a list for SaaS software. How does it qualify ?

Reply:
Yeah. My mistake. I'll delete the PR

Reply:
Thank you